script/openqa-clone-job

```diff
@@ -110,9 +110,9 @@ from JOBREF if it is a URL).
 Specifies the directory to store test assets (defaults to
 $OPENQA_SHAREDIR/factory).
 
-=item B<--skip-checks>
+=item B<--check-repos>
 
-Skips additional checks like maintenance update availability.
+Additionally check maintenance update repositories availability(SUSE specific check).
 
 =item B<--skip-deps>
```

Comment:
missed

Reply:
I don't understand . what does it mean ? what exactly missed ?